src/main/java/frc/robot/RobotContainer.java

```diff
@@ -681,29 +681,29 @@ public Command scoreCoral() {
                 elevator.setHeight(ElevatorConstants.STOW_METER).withTimeout(0.0),
                 endEffector.stopMotor())
             .withTimeout(0.5);
-
+    // start here
     Command deAlgae =
         Commands.sequence(
                 endEffector.shoot(EndEffectorConstants.VOLTAGE_L4).onlyIf(IS_L4),
                 endEffector.shoot(EndEffectorConstants.VOLTAGE_L3).onlyIf(IS_L3),
                 endEffector.shoot(EndEffectorConstants.VOLTAGE_L2).onlyIf(IS_L2),
                 endEffector.shoot(EndEffectorConstants.VOLTAGE_L1).onlyIf(IS_L1),
-                Commands.waitSeconds(0.05),
+                Commands.waitSeconds(0.04), // .05
```

Comment:
This is 40 milliseconds - which is basically two ticks of the command loop.  Each is 20 milliseconds, so 50 milliseconds would round up to 3 cycles, roughly.